journal-letters/original/07-chapter-7.adoc

```diff
@@ -198,7 +198,7 @@ And so every day high praises may be in your hearts and mouths unto her King,
 who is King of kings, and reigns over all, who only hath immortality,
 and dwells in the light: to whom be glory and eternal praises,
 saith my soul for evermore.
-Amen.
+Amen. From your friend, and one that truly loves you in the Truth.
```

Comment:
This phrase is already in the closing below, (which is what we're changing the modernized edition to match), so we need to remove it from this line so it's not doubled.